src/resources/LimitedRestorationToFull/BC.ts

```diff
@@ -27,6 +27,7 @@ export const RestorationResourceBc: ResourceIF = {
       subtitle: null // not used
     },
     nameRequestTypes: [
+      NrRequestActionCodes.CHANGE_NAME,
       NrRequestActionCodes.RESTORE
```

Comment:
If the business is in limited restoration then it's active, and Namerequest UI won't find it in a search for a restoration NR. So, this allows the use of a change of name NR for a limited-restored business.